typhon-webapp/src/pages/add_project.rs

```diff
@@ -0,0 +1,215 @@
+            padding-top: 22px;
+            max-width: 600px;
+            margin: auto;
+        }
+        .fields {
+            display: flex;
+            flex-direction: column;
+            gap: 14px;
+        }
+        .field > label {
+            display: block;
+            padding-left: 2px;
+            font-size: var(--font-size-small);
+            font-weight: 400;
+        }
+        .field > input {
+            width: 100%;
+        }
+        .field.flake {
+            display: flex;
+            flex-direction: column;
+            gap: 16px;
+        }
+        .field.flake .option {
+            display: grid;
+            grid-template-areas: raw_str("radio icon title") raw_str("radio icon details");
+            grid-template-columns: auto auto 1fr;
+            align-items: center;
+            column-gap: 5px;
+        }
+        .field.flake .option input {
+            grid-area: radio;
+        }
+        .field.flake :deep(svg) {
+            grid-area: icon;
+            font-size: 20px;
+        }
+        .field.flake .option .kind {
+            font-size: var(--font-size-small);
+            font-weight: 400;
+            grid-area: title;
+            padding-bottom: 2px;
+        }
+        .field.flake .option .details {
+            font-size: var(--font-size-small);
+            grid-area: details;
+            color: var(--color-fg-muted);
+            padding-top: 2px;
+        }
+        button {
+            background: var(--color-green-button-bg);
+            color: white;
+            width: auto!important;
+            transition: all 100ms;
+        }
+        button:hover {
+            transition: all 100ms;
+            background: var(--color-green);
+        }
+        button:active {
+            transition: all 100ms;
+            background: var(--color-darkgreen);
+        }
+        .page :deep(.message) {
+            text-align: justify;
+            padding-bottom: 14px;
+            display: flex;
+            align-items: center;
+            gap: 6px;
+        }
+        .page :deep(.error-message) {
+            color: var(--color-danger-emphasis);
+        }
+        .page :deep(.success-message) {
+            color: var(--color-success);
+        }
+        // TODO: Move to `app.rs`
+        .page :deep(button) {
+            border-radius: 6px;
+            border: 1px solid var(--color-border-default);
+            padding: 8px 10px;
+            cursor: pointer;
+            outline: inherit;
+            font-size: inherit;
+            font-family: inherit;
+            font-weight: 400;
+        }
+    };
+    view! { class=style,
+        <div class="page">
+            <ActionForm action>
+                <div class="header">"Add a project"</div>
+                <div class="header-description sep">
+                    "Add continuous integration with Typhon for an existing codebase. This project will be visible to anyone that have access to this Typhon instance."
```

Comment:
How about "Import a project's declaration to Typhon. This project will be visible to anyone with access to this Typhon instance."?